CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ mapView.location2.puckBearingEnabled = false
 * Fix location puck not being shown if map is created without initial style (e.g. MapInitOptions.styleUri == null) and then loaded asynchronously. ([#1114](https://github.com/mapbox/mapbox-maps-android/pull/1114))
 * Fix crash within location plugin that happens when style is reloaded simultaneously with location plugin updates. ([#1112](https://github.com/mapbox/mapbox-maps-android/pull/1112))
 * Fix memory leak in location component. ([#1093](https://github.com/mapbox/mapbox-maps-android/pull/1093))
+* Fix possible memory leaks in location component. ([#1172](https://github.com/mapbox/mapbox-maps-android/pull/1172))
```

Comment:
can we simply add ref to #1172 to an entry in line 41?

Reply:
Though about it, will join both